packages/storage/src/indexed-db/indexed-db.test.ts

```diff
@@ -94,7 +99,9 @@ describe('IndexedDb', () => {
         chainId: 'test',
         accountAddr: 'penumbra123xyz',
         dbVersion: 2,
-        walletId: `walletid${Math.random()}`,
+        walletId: new WalletId({
+          inner: Uint8Array.from({ length: 8 }, () => Math.floor(Math.random() * 256)),
+        }),
```

Comment:
This test is actually incorrect, if we want to test idb version update, we should not generate a random dbName, only increase the version.
But the thing is, if we fix this test, it causes all storage tests to hang and timeout error. I will open a separate PR for this fix (since this PR is big without it)